calblueprint/siren
# combine doc container and doc upload

# ✨ New in this PR ✨

## One liner: what did you do? 🚨 
Combined doc container PR and doc upload PR

## Coverage 🙆‍♀️
_use this section to break up your task into submodules and track progress_ 
 - [x] Refactor doc container PR to be compatible with doc upload 
 - [x] Create documents in the case after clicking upload
 - [ ] Speed up upload (todo listed in code) 

## How can the reviewer test your code? 👩‍💻
_example: run `npm start` and click through Navigation bar_ 

1. go to upload screen
2. click on a document
3. upload something
4. ensure that it is checked on the doccontainer list
5. go to firestore, ensure client/case/document was created
6. go to storage, ensure file was uploaded correctly

## Any bugs you encountered or still having trouble with? 🐛
_describe how you solved them such that someone can follow your explanation_ 


## Resources 📔
_link online resources and related PRs_ 


🧜‍♀️ cc: @shannonbonet @phoebeli23
